### DISCUSSION TITLE
QUICTLS based on OSSL 3.3

### DISCUSSION THREAD

**Discussion by samitanwer** [quictls/openssl#170] [General]
The branch [quic-on-3.3-dirty](https://github.com/quictls/openssl/tree/wladd/quic-on-3.3-dirty) has not been touched lately. Is there anything blocking adoption of OSSL 3.3?  🔎🔎

**Comment by richsalz** [comment]
We are not doing openssl 3.3, instead we have made the fork quictls/quictls based on 3.3.
